apisix/plugin.lua

```diff
@@ -907,7 +907,7 @@ local function enable_gde()
     if enable_data_encryption == nil then
         enable_data_encryption =
             core.table.try_read_attr(local_conf, "apisix", "data_encryption",
-                    "enable_encrypt_fields")
+                    "enable_encrypt_fields") and (core.config.type == "etcd")
```

Comment:
encrypt fields only when config type is etcd. Without this addition, enabling `encrypt_fields` on standalone mode would cause unexpected failures.